src/interfaces/problem.result.interface.ts

```diff
@@ -0,0 +1,9 @@
+
+  enum ProblemResultType{PRELIMINARY='PRELIMINARY', FINAL=' FINAL'}
```

Comment:
Be careful of the space in the beginning of the value ` FINAL` this will cause it to fail.